guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -143,6 +143,61 @@ To edit the WAF error page:
 
    -  On the Cache Management page, click **Flush Magento Cache**.
 
+## Display Magento error report number
+
+By default, Fastly hides all Magento errors behind the _503 Service Unavailable_ error. To display the Magento error log report number so that you can find and review the error details in the logs, open the website omitting Fastly using these steps:
+
+1. Retrieve the IP address of your store:
+
+   -  Pro plan users, Staging and Production environments:
+
+      ```bash
+      nslookup {your_project_id}.ent.magento.cloud
+      ```
+
+   -  Pro plan users, Integration environment; Starter plan users, all environments:
+
+      ```bash
+      nslookup gw.{your_region}.magentosite.cloud
+      ```
+
+1. Add your application domain and IP address to the hosts file on your local machine:
```

Comment:
We don't use colons at the end of a step, even with a command.
Also, not sure why using "local machine".
We already have a problem keeping consistent between "local workstation" and "local workspace". Need to stick with one of them.

Reply:
We do use colons at the end of steps, but I noticed that you have been removing them.  I think when the step is followed by a command that is carrying out the step, a colon makes sense.  I wonder what the style guide says.  The style guide doesn't say anything.  Google style guide procedure shows colon as recommended in the context of a specific example. 

I notice that this convention of using the colon is followed in other DevDoc Guides.